python/cudf/bindings/cudf_cpp.pxd

```diff
@@ -222,9 +222,14 @@ cdef extern from "cudf.h" nogil:
 
     cdef gdf_error gdf_validity_and(gdf_column *lhs, gdf_column *rhs, gdf_column *output) except +
 
-    cdef gdf_size_type gdf_dtype_size(gdf_dtype dtype) except +
-
-    cdef gdf_error get_column_byte_width(gdf_column * col, int * width) except +
+    cdef gdf_error gdf_filter(size_t nrows,
```

Comment:
`gdf_filter` was removed. It looks like this was mistakenly added back during a merge. You can remove the `gdf_filter` from this file.